auto_round/autoround.py

```diff
@@ -939,10 +966,11 @@ def quant_block(self, block, input_ids, input_others, q_input=None, device=torch
                     seqlen=self.seqlen,
                     share_attention_mask_flag=self.share_attention_mask_flag,
                     not_share_position_ids_flag=self.not_share_position_ids_flag,
+                    not_share_rotary_pos_emb_flag=self.not_share_rotary_pos_emb_flag,
                     input_dim=self.input_dim,
                 )
 
-                current_output = [output[i] for i in indices]
+                current_output = [output[x] for x in indices]
```

Comment:
why use x

Reply:
To distinguish it from the index i of the for loop and avoid potential problems